plugins/providers/maxcompute/credentials.go

```diff
@@ -7,11 +7,12 @@ import (
 )
 
 type credentials struct {
-	AccessKeyID     string `mapstructure:"access_key_id" json:"access_key_id"`
-	AccessKeySecret string `mapstructure:"access_key_secret" json:"access_key_secret"`
-	RAMRole         string `mapstructure:"ram_role" json:"ram_role"`
-	RegionID        string `mapstructure:"region_id" json:"region_id"`
-	ProjectName     string `mapstructure:"project_name" json:"project_name"`
+	AccessKeyID     string   `mapstructure:"access_key_id" json:"access_key_id"`
+	AccessKeySecret string   `mapstructure:"access_key_secret" json:"access_key_secret"`
+	RAMRole         string   `mapstructure:"ram_role" json:"ram_role"`
+	RegionID        string   `mapstructure:"region_id" json:"region_id"`
+	ProjectName     string   `mapstructure:"project_name" json:"project_name"`
+	ExcludedSchemas []string `mapstructure:"excluded_schemas" json:"excluded_schemas"`
```

Comment:
in the `ProviderConfig.Resources[]` config, we already have `filter` field that uses expression to exclude resources. Can we utilize the same config instead of introducing a new one? The implementation should be similar to how we handle dataset filtering in BQ